src/test/java/com/commercetools/sync/inventory/utils/InventoryDraftTransformerTest.java

```diff
@@ -0,0 +1,93 @@
+    private static final String CUSTOM_FIELD_VALUE = "testValue";
+
+    @Test
+    public void transformToDraft_WithNoChannelNeitherCustomType_ShouldReturnDraft() {
+        final InventoryEntry entry = InventoryEntryMock
+                .of(SKU, QUANTITY_ON_STCK, RESTOCKABLE_IN_DAYS, EXPECTED_DELIVERY).build();
+        final InventoryEntryDraft draft = InventoryDraftTransformer.transformToDraft(entry);
+
+        assertThat(draft.getSku()).isEqualTo(SKU);
+        assertThat(draft.getQuantityOnStock()).isEqualTo(QUANTITY_ON_STCK);
+        assertThat(draft.getRestockableInDays()).isEqualTo(RESTOCKABLE_IN_DAYS);
+        assertThat(draft.getExpectedDelivery()).isEqualTo(EXPECTED_DELIVERY);
+        assertThat(draft.getCustom()).isNull();
+        assertThat(draft.getSupplyChannel()).isNull();
+    }
+
+    @Test
+    public void transformToDraft_WithSupplyChannel_ShouldReturnDraft() {
+        final InventoryEntry entry = InventoryEntryMock.of(SKU)
+                .withChannelRefExpanded(CHANNEL_ID, CHANNEL_KEY).build();
+        final InventoryEntryDraft draft = InventoryDraftTransformer.transformToDraft(entry);
+
+        assertThat(draft.getSupplyChannel()).isNotNull();
+        assertThat(draft.getSupplyChannel().getId()).isEqualTo(CHANNEL_ID);
+        assertThat(draft.getSupplyChannel().getObj()).isNotNull();
+        assertThat(draft.getSupplyChannel().getObj().getKey()).isEqualTo(CHANNEL_KEY);
+    }
+
+    @Test
+    public void transformToDraft_WithCustomTypeWithoutKey_ShouldReturnDraft() {
+        final InventoryEntry entry = InventoryEntryMock.of(SKU)
+                .withCustomField(CUSTOM_TYPE_ID, CUSTOM_FIELD_NAME, CUSTOM_FIELD_VALUE).build();
+
+        final InventoryEntryDraft draft = InventoryDraftTransformer.transformToDraft(entry);
+
+        assertThat(draft.getCustom()).isNotNull();
+        assertThat(draft.getCustom().getType()).isNotNull();
+        assertThat(draft.getCustom().getType().getId()).isEqualTo(CUSTOM_TYPE_ID);
+        assertThat(draft.getCustom().getFields()).isNotNull();
+        assertThat(draft.getCustom().getFields().containsKey(CUSTOM_FIELD_NAME)).isTrue();
+    }
+
+    @Test
+    public void transformToDraft_WithCustomTypeWithKey_ShouldReturnDraft () {
+        final InventoryEntry entry = InventoryEntryMock.of(SKU)
+                .withCustomFieldExpanded(CUSTOM_TYPE_ID, CUSTOM_TYPE_KEY, CUSTOM_FIELD_NAME, CUSTOM_FIELD_VALUE)
+                .build();
+
+        final InventoryEntryDraft draft = InventoryDraftTransformer.transformToDraft(entry);
+
+        assertThat(draft.getCustom()).isNotNull();
+        assertThat(draft.getCustom().getType()).isNotNull();
+        assertThat(draft.getCustom().getType().getKey()).isEqualTo(CUSTOM_TYPE_KEY);
+        assertThat(draft.getCustom().getFields()).isNotNull();
+        assertThat(draft.getCustom().getFields().containsKey(CUSTOM_FIELD_NAME)).isTrue();
+    }
+
+    @Test
+    public void transformToDrafts_ShouldReturnDrafts() {
```

Comment:
The test name and asserts are confusing. From the name it is expected to test returned drafts, but only the number of drafts is tested.